draft-dnsop-deleg.md

```diff
@@ -149,60 +147,23 @@ NS based delegation supports DNS over UDP and TCP, but does not have the ability
 
-## Introductory Examples
-
-To introduce DELEG record, this example shows the authority section of a DNS response that delegates a subdomain to another nameserver.
-
-
-    example.com.  86400  IN DELEG  1 ns1.example.com. (
-                    ipv4hint=192.0.2.1 ipv6hint=2001:DB8::1 )
-    example.com.  86400  IN NS     ns1.example.com.
-    ns1.example.com.    86400   IN  A  192.0.2.1
-    ns1.example.com     86400   IN  AAAA    2001:DB8::1
-
-In this example, the authoritative nameserver is delegating using the same parameters as regular DNS, but the delegation as well as the glue can be signed.
-
-Like in SVCB, DELEG also offer the ability to use the Alias form of delegation. The example below shows an example where example.com is being delegated with a DELEG AliasMode record which can then be further resolved using standard SVCB to locate the actual parameters.
-
-    example.com.  86400  IN DELEG 0   config2.example.net.
-    example.com.  86400  IN NS     ns2.example.net.
-
-The example.net authoritative server may return the following SVCB records in response to a query as directed by the above records.
-
-    config2.example.net 3600    IN SVCB . (
-                    ipv4hint=192.0.2.54,192.0.2.56
-                    ipv6hint=2001:db8:2423::3,2001:db8:2423::4 )
-
-The above records indicate to the client that the actual configuration for the example.com zone can be found at config2.example.net
-
-Later sections of this document will go into more detail on the resolution process using these records.
-
-## Goal of the DELEG record
-
-The primary goal of the DELEG records is to provide zone owners a method to signal capabilities to clients how to connect and validate a subdomain. This method coexists with NS records in the same zone. 
-
-The DELEG record is authoritative in the parent zone and, if signed, has to be signed with the key of the parent zone. The target of an alias record is an SVCB record that exists and can be signed in the zone it is pointed at, including the child zone.
-
-## DNSSEC is RECOMMENDED
-
-While DNSSEC is RECOMMENDED, unsigned DELEG records may be retrieved in a secure way from trusted, Privacy-enabling DNS servers using encrypted transports.
-
-FOR DISCUSSION: This will lead to cyclical dependencies. A DELEG record can introduce a secure way to communicate with trusted, Privacy-enabling DNS servers. For that, it needs to be DNSSEC signed. 
+# DELEG Record Type
 
-### Preventing downgrade attacks
+The DELEG record wire format is the same as the SVCB record defined in {{?RFC9460}}, but with a different value for the resource record type of TBD. The IANA registry for for further Service Parameter Keys that might be needed also is the one used for SVCB.
```

Comment:
nit: 'IANA registry **for for** further Service'
The sentence does not parse. I feel like something has gone missing.

Reply:
That section has been re written and should be clearer now.